cmd/kops/rollingupdatecluster.go

```diff
@@ -264,5 +271,10 @@ func RunRollingUpdateCluster(f *util.Factory, out io.Writer, options *RollingUpd
 		return nil
 	}
 
+	if options.Headroom > 0 {
+		fmt.Printf("\nkops will not apply headroom to master and bastion nodes\n")
+		fmt.Printf("Warning: kops will not terminate extra nodes added as headroom for rolling updates\n")
```

Comment:
If you start with 5 nodes with --headroom=3, you're final node will be 8 and not 5 (that seemed like the safer route, not sure if its correct)